ci/buildspec-extension.yml

```diff
@@ -0,0 +1,26 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      docker: 18
+      python: 3.7
+  pre_build:
+    commands:
+    - echo Logging in to Amazon ECR...
+    - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION)
+  build:
+    commands:
+    - echo Build started on `date`
+    - echo Building the Docker image...
+    - docker build -t sklearn-base:0.21.2-cpu-py3 -f docker/0.21.2/base/Dockerfile.cpu .
+    - pip install wheel setuptools
+    - python setup.py bdist_wheel
+    - docker build -t preprod-sklearn:0.21.2-cpu-py3 -f docker/0.21.2/final/Dockerfile.cpu .
```

Comment:
I would create a buildspec env for the version and replace everywhere, instead of copy pasting within the file.